packages/theme/src/cli/utilities/theme-command.ts

```diff
@@ -157,10 +153,13 @@ export default abstract class ThemeCommand extends Command {
       }
```

Comment:
A check to resolve `path` values was added to [this PR](https://github.com/Shopify/cli/pull/6361/files#diff-5d3425a5a7069d7e5072b07c9a55347067bacd01536cc3ea6e9d101e95893262R139)